### ISSUE TITLE
adding feature fails, object was destroyed

### ISSUE THREAD

**Issue by sbowler** [openlayers/ol3-cesium#177]
I seem to be having a lot of issues trying to test this out in an already working ol3 application. I have an openlayers angular controller that handles all mapping. This controller contains the ol map object and all of the layers. I added the 3d map initialization right after the 2d map initialization per the examples and readme. Even without enabling the 3d map I instantly seem to have issues adding a feature to the map which causes it to not even get added to the 2d map. I believe the behind the scenes synchronization is breaking and the feature to not be added.

I'm not very familiar with Cesium, but it appears to be throwing this exception.
 <code>{name: "DeveloperError", message: "This object was destroyed, i.e., destroy() was called.",</code>

So far as I can tell it is saying the PrimitiveCollection in cesium that should store the collection of those layers was destroyed for some reason? I add a lot of extra properties to the features so I tried removing those to no avail. I also tried creating another very basic vector layer much like the simple examples that doesn't have as much going on with it, like changing the sources and such. That seemed to fail just the same as well. 

In short I'm basically just doing a <code>vectorSource.addFeature(newMarker);</code>
where newMarker is a ol.Feature and that vector source belongs to a vector layers that's added to the map.

Unfortunately the code is proprietary so I can't really share it. Any ideas would be appreciated?
